test/k8sT/DatapathConfiguration.go

```diff
@@ -610,6 +610,15 @@ var _ = Describe("K8sDatapathConfig", func() {
 		})
 	})
 
+	Context("Debug", func() {
+		It("Check connectivity with debugging of the BPF compilation process enabled", func() {
+			deploymentManager.DeployCilium(map[string]string{
+				"debug.verbose": "datapath-compile",
+			}, DeployCiliumOptionsAndDNS)
+			Expect(testPodConnectivityAcrossNodes(kubectl)).Should(BeTrue(), "Connectivity test between nodes failed")
```

Comment:
Should we look for a specific debug output in Cilium logs in this test?

Reply:
Good shout :+1: 
Yes, I think I can look for `Enabling BPF compilation process debug` to confirm `--debug-verbose=datapath-compile` is actually set